apps/SageAccountWeb/src/style/theme.ts

```diff
@@ -112,7 +73,6 @@ export const themeOptions = mergeTheme({
       fontFamily: latoFont,
       fontSize: '14px',
     },
-    headline1: {},
```

Comment:
Removed overrides that were empty or identical to what is defined in synapse-react-client